rel/overlay/bin/remsh

```diff
@@ -88,6 +89,9 @@ while getopts ":hn:c:l:mv" optionName; do
     v)
       VERBOSE=0
       ;;
+    t)
```

Comment:
I think we need a `t:` in the getops string for this to work.

Reply:
added